scripts/utils/price_utils.js

```diff
@@ -198,8 +164,8 @@ const getOneinchPrice = async function (baseToken, quoteToken, globalPriceStorag
   return priceFeedSlice
 }
 
-const isPriceReasonable = async (baseTokenData, quoteTokenData, price, acceptedPriceDeviationInPercentage = 2) => {
-  const onlinePriceSlice = await getOneinchPrice(quoteTokenData, baseTokenData)
+const isPriceReasonable = async (buyToken, sellToken, price, acceptedPriceDeviationInPercentage = 2) => {
```

Comment:
I really like base and quote here. Base and quote describe much better the expected price than buy and sell token.

Reply:
I have put them back.